next.config.js

```diff
@@ -10,6 +10,11 @@ const nextConfig = {
           '/near/widget/NearOrg.HomePage?utm_source=stack&utm_medium=podcast&utm_campaign=stackoverflow_evergreen_bos_awareness',
         permanent: false,
       },
+      {
+        source: '/nearcon23.near/widget/Index',
+        destination: 'https://nearcon.org',
```

Comment:
`nearcon.org` automatically redirects to `nearcon.app`, should we redirect straight there instead?

Reply:
I just noticed that.  Hmm.. I suppose so.